master.js

```diff
@@ -927,70 +924,64 @@ class wsSlave {
 		if(commandID) this.commandsWaitingForReturn[commandID] = {callback, timestamp: Date.now()};
 	}
 }
+
 io.on('connection', function (socket) {
-	// cleanup dead sockets from disconnected people
```

Comment:
Everything else in this file uses tabs, was this change to spaces necessary?

Reply:
not intentional. just the ide doing I guess